src/system_data.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python2
+
+import rospy
+import subprocess
+from std_msgs.msg import String
+
+def dataRead():
+    
+    ps_pub = rospy.Publisher('ps_stats', String, queue_size=20)
+    top_pub = rospy.Publisher('top_stats', String, queue_size=20)
+    tegra_pub = rospy.Publisher('tegra_stats', String, queue_size=20)
+    rospy.init_node('dataRead')
```

Comment:
This name needs to be changed, can we replace `'dataRead'` with `'system_stat_monitor'`?

Reply:
And move the call to `init_node` up before the publishers are created?

Reply:
Sure thing.

Reply:
What is the difference between calling 'init_node' before and after?

Reply:
In some cases it can invalidate publishers and subscribers created before `init_node` was called.  That wouldn't happen in this case, but it's better practice to call `init_node` first.